### DISCUSSION TITLE
[Question][Hw2] About latesubmission policy

### DISCUSSION THREAD

**Discussion by JOYB28** [prosyslab-classroom/cs348-information-security#692] [Q&A]
Name: Yoonseo Kim

Is it possible to consider the past record(which is not late) after late submission?

For example, before late submission my point in 200.
And after 1-day late submission, my point still 200. And then my final result will be 160(200 * 80%) or 200?

**Comment by KihongHeo** [comment]
Hi.
- Unfortunately, we will only consider the latest submission. Otherwise, things get too complicated (e.g., a lot of game-playing may happen). Let us keep it simple as much as possible.
- That is life. Everyone should take responsibility for the action. (낙장불입)
- Do not get frustrated by the penalty this time. If you worked really hard, you will get the credit in another form later in your life. I am sure.

**Comment by tncks0121** [comment]
In Gradescope I can "activate" which submission to be graded, not only the last submission. Is that feature non-related and only the last submission is selected for grading? 

I'm not sure because lecture1.pdf says the final commit of the main branch is used for grading, so according to that, it seems even the last submission on Gradescope is not related

**Comment by KihongHeo** [comment]
I don't think it is a good idea to make things complicated.

We provide Gradescope leaderboard for your convenience. Also late submissions are allowed for your convenience. I.e., Students get feedback immediately and improve their scores. 

Do not abuse them. We only use the latest submission in Gradescope.

**Comment by tncks0121** [comment]
Thanks for the clarification. I wanted to be sure that  if I get scores like 285 / 280 / 270 in order of submissions, I must resubmit the 285 to get the best score, not just activating 285.

**Comment by JOYB28** [comment]
Thanks for explanation. I understood!
